dotCMS/src/main/java/com/dotmarketing/portlets/workflows/util/WorkflowSchemeImportExportObject.java

```diff
@@ -20,15 +20,24 @@
 	List<WorkflowScheme> schemes;
 	List<WorkflowStep> steps;
 	List<WorkflowAction> actions;
+	List<Map<String, String>> actionSteps;
 	List<WorkflowActionClass> actionClasses;
 	List<WorkflowActionClassParameter> actionClassParams;
 	List<Map<String, String>> workflowStructures;
-	
-	
-	
-	
 
 
+	public List<Map<String, String>> getActionSteps() {
+		if(actionSteps != null){
+			return actionSteps;
+		}
+
+		return new ArrayList<Map<String,String>>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)